app/assets/stylesheets/govuk_publishing_components/components/govspeak/_charts.scss

```diff
@@ -24,13 +24,17 @@
   $stack-spacing: 1px; // For stacked bars, the spacing between each stack in a bar
   $caption-side: top; // Caption alignment. Top or bottom.
   $key-width: 160px; // Only used by IE. Other browsers get smallest width that fits content
+  $dark-blue: #12436d;
```

Comment:
A comment to explain why we're not using standard govuk-colours would be good here to avoid somebody helpfully reverting this in future.

Reply:
I had the same thoughts on this, was thinking we could even prefix add a `gss` prefix to the variables as well...

```SCSS
$gss-dark-blue: #12436d;
$gss-orange: #f46a25;
$gss-dark-pink: #801650;
$gss-turquoise: #28a197;
```

Reply:
Thanks both, makes sense have added comments and amended the variable names too.